cogs/audio.py

```diff
@@ -1727,6 +1777,8 @@ def has_connect_perm(self, author, server):
                 song = None
             self.queue[server.id]["NOW_PLAYING"] = song
             log.debug("set now_playing for sid {}".format(server.id))
+            await self._update_bot_status()
```

Comment:
I'm gonna suggest `loop.create_task()` here too because this isn't essential to `await`. It'll just slow down the loop